nilearn/datasets/description/spm_auditory.rst

```diff
@@ -0,0 +1,39 @@
+SPM auditory
+
+Notes
+-----
+These whole brain BOLD/EPI images were acquired on a modified 2T Siemens MAGNETOM Vision system.
+Each acquisition consisted of 64 contiguous slices (64x64x64 3mm x 3mm x 3mm voxels).
+Acquisition took 6.05 seconds, with the scan to scan repeat time (RT) set arbitrarily to 7 seconds.
+
+96 acquisitions were made (RT= 7 seconds), in blocks of 6, giving 16 blocks of 42 seconds.
+The condition for successive blocks alternated between rest and auditory stimulation,
+starting with rest.
+
+Auditory stimulation was bi-syllabic words presented binaurally at a rate of 60 per minute.
+
+A structural image was also acquired.
+
+This experiment was conducted by Geriant Rees
+under the direction of Karl Friston and the FIL methods group.
+
+Content
+-------
+        - 'func': :obj:`list` of :obj:`str`. Paths to functional images
+        - 'anat': :obj:`list` of :obj:`str`. Path to anat image
+        - 'description': :obj:`str`. Data description
```

Comment:
can this indent match the others or is it related to the way it renders?

Reply:
I had actually not finished cleaning up this one. Thanks for catching it.

Those description don't render anywhere for now, so there may be a lot of clean up to do when start rendering them.